### DISCUSSION TITLE
Need clarification regarding plasma-manager and plasma addons

### DISCUSSION THREAD

**Discussion by SpidFightFR** [nix-community/plasma-manager#409] [Q&A] [answered]
Hey guys, i had a question about plasma-manager.
Are the items specified in a plasma-manager config (E.g: icontheme, plasma themes, etc) downloaded by the plasma-manager module, using kde discover, if they're not present in the base system (in case of a new installation, for example) ?
Or should we make sure they are available beforehand ?

**Comment by HeitorAugustoLN** [comment, marked as answer]
Everything that you add on top of plasma needs to be installed manually with their respective package. The only thing we can automatically install are the third party widget packages that we provide modules for. That are almost every packaged plasma widget available (with some exceptions). But everything else such as the themes, fonts, icon themes and cursor themes, they need to be installed manually to be set with plasma-manager.

**Reply by SpidFightFR** [reply]
Okay, thank you !
